### ISSUE TITLE
create domain error when setting default realm in wls10.3.6

### ISSUE THREAD

**Issue by 53d117460ec63d70** [oracle/weblogic-deploy-tooling#296]
Create domain fails when setting default realm. Extract from my model after domain discover:

```yaml
    SecurityConfiguration:
        DefaultRealm: robrealm
        Realm:
            robrealm:
                Adjudicator:
                    DefaultAdjudicator:
                        DefaultAdjudicator:
                AuthenticationProvider:
                    DefaultAuthenticator:
                        DefaultAuthenticator:
                    DefaultIdentityAsserter:
                        DefaultIdentityAsserter:
                            ActiveType: [ AuthenticatedUser ]
                Authorizer:
                    XACMLAuthorizer:
                        XACMLAuthorizer:
                            PolicyDeploymentEnabled: true
                CertPathProvider:
                    WebLogicCertPathProvider:
                        WebLogicCertPathProvider:
                CredentialMapper:
                    DefaultCredentialMapper:
                        DefaultCredentialMapper:
                RoleMapper:
                    XACMLRoleMapper:
                        XACMLRoleMapper:
```
I get the error:
```<WLSDPLY-12409> <createDomain failed to create the domain: Server robrealm not found in domain or resource group>```

If I remove `DefaultRealm: robrealm` from my model the domain is successfully created. However, two security domains are then created, `robrealm`  and `myrealm`, and `myrealm` is set as the default realm.

**Comment by rakillen** [comment]
This was fixed with #282 , and will be in the next release

**Comment by 53d117460ec63d70** [comment]
OK, thanks.
